core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -81,17 +99,29 @@ private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolea
     new File(subDir, filename)
   }
 
-  def getFile(blockId: BlockId): File = getFile(blockId.name)
+  /**
+   * Used only for testing.
```

Comment:
This is wrong because this is used in this class.
Do you want to write "Visible for testing"?

Reply:
See my above comments.